lib/DDG/Spice/Polls.pm

```diff
@@ -0,0 +1,42 @@
+package DDG::Spice::Polls;
```

Comment:
I think `Spice::ElectionPolls` or `Spice::HuffPostPolls` is a little better? I'm worried "Polls" is a little too generic